### ISSUE TITLE
Reward broadcast not working

### ISSUE THREAD

**Issue by Artuto** [BenCodez/VotingPlugin#440]
Hi, I'm having an issue where I have
`Reward: 'Default'` in my config file, where Default is a file called Default.yml in the Rewards folder.

However, the first line is a `BroadcastMsg: '...'` and the next one is `Money: 2000`.

The money is added to my account, but the Broadcast isnt made.

**Comment by BenCodez** [comment]
It's something like this
Messages:
  Broadcast: ''

Look in the example basic reward

**Comment by Artuto** [comment]
Yay that worked. However, now I'm having another issue, when a player joins they get the reward even if they didnt voted.

This only happens once with players that never logged after the plugin was installed.

**Comment by Artuto** [comment]
Update: doesn't happens only once, its every time they join. Please help ASAP

**Comment by Artuto** [comment]
Nvm got it fixed.
